### PR TITLE
Promote/Azure Container App deployment example

### PR DESCRIPTION
**Pull request by black-bryan**
Promotion of the scim-example for Azure Container App SCIM bridge deployment from the "Beta" examples directory.

- Small changes to readme to align with other scim-examples formatting of code blocks, etc.
- Alignment of readme content for template deployment method using Azure Cloud Shell or `az` CLI tools. 

This example has been used by many customers since it was first added to the Beta directory.

### CODE REVIEW

File: azure-container-apps/ReadMe.md [hunk 1]
```diff
@@ -51,33 +51,29 @@ Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefit
...
 
 Both methods need the Container App Extension added to the Azure tool of choice, as detailed in step two.
 
 1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com) or directly open the [Azure Shell](https://shell.azure.com).
 2. Add the Azure Container App extension by running the following command: `az extension add --name containerapp --upgrade`
-3. Get a list of the available locations in your Azure account: `az account list-locations -o table`, verify that the region you want to deploy on [supports Azure Container Apps](https://azure.microsoft.com/en-ca/explore/global-infrastructure/products-by-region/?regions=all&products=container-apps), take note of the name field for the desired region. 
+3. Get a list of the available locations in your Azure account: `az account list-locations -o table`, take note of the name field for the desired region. 
 
     > **Note:** This command can be skipped if you already know the region you want to deploy to and if that region supports Container Apps.
```
Comment (accraw) [inline azure-container-apps/ReadMe.md:64]: Do all regions now support container apps? If so, this note should be updated too.

Comment (black-bryan) [reply azure-container-apps/ReadMe.md:64]: Not all regions support Container Apps still. The amount of regions has increased drastically. The wording that will end up in AKB is slightly different, and once that ends up in AKB, this section with the deployment steps will be removed from this readme.

"Find your preferred server location (region) for the deployment and make note of its `Name`, such as `eastus`." is the wording that will be used and no mention in AKB about the supported regions. This is why we have added a section to readme in [Troubleshooting for Region Support](https://github.com/1Password/scim-examples/tree/promote/azure-container-apps/azure-container-apps#region-support).

File: azure-container-apps/ReadMe.md [hunk 2]
```diff
@@ -534,23 +498,3 @@ Logs for a container app can be viewed from the **Log Stream**, there is a separ
 7. Test logging into your SCIM bridge URL with the new `bearer token`.
 8. Update your identity provider configuration with the new bearer token.
 </details>
-
-## Update 1Password SCIM Bridge
```
Comment (accraw) [inline azure-container-apps/ReadMe.md:538]: Where did this get moved to?

Comment (black-bryan) [reply azure-container-apps/ReadMe.md:538]: [Line 250 - 272](https://github.com/1Password/scim-examples/blob/562c4fdcc8d8e7a5e2406a715d0b83b6cd264bb2/azure-container-apps/ReadMe.md?plain=1#L250-L272)
I was second guessing myself, as it wasn't seeing it to being with either.

**Review by accraw — APPROVED** [review]

**Pull request merged**